.travis.yml

```diff
@@ -0,0 +1,33 @@
+os: linux
+sudo: false
+language: perl
+perl:
+    - "5.26"
+before_install:
+    - cpanm Carton
+install:
+  - "cd Conch; carton install"
```

Comment:
Can this use --deployment, so we're installing the versions in the lockfile?

Reply:
Not until String::CamelCase fixes its META file